templates/download/xilinx/index.html

```diff
@@ -31,78 +31,293 @@ <h1>Install Ubuntu on Xilinx</h1>
-      {{ image (
-        url="https://assets.ubuntu.com/v1/24e23fbe-zcu106-evaluation-board.png",
-        alt="",
-        width="400",
-        height="273",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
-
 <section class="p-strip">
-  <div class="row u-vertically-center">
-    <div class="col-6">
-      <h3 class="p-heading--4">First time installing Ubuntu on Xilinx?</h3>
-      <p>Please see the <a class="p-link--external" href="https://xilinx-wiki.atlassian.net/wiki/spaces/A/pages/2037317633/Getting+Started+with+Certified+Ubuntu+20.04+LTS+for+Xilinx+Devices?utm_source=canonical&utm_medium=referral&utm_campaign=ubuntu&utm_content=download&utm_term=desktop_image">Xilinx Getting Started Guide</a> for more information.</p>
-      <p><strong>Xilinx Ubuntu Software Development Kit</strong></p>
-      <p>Ubuntu Developers targeting Xilinx Zynq UltraScale+ MPSoC devices may also want to download the following files:</p>
-    </div>
-  </div>
   <div class="u-fixed-width">
-    <table aria-label="Xilinx Ubuntu Software Development Kit table">
-      <thead>
-        <tr>
-          <th>Image name</th>
-          <th colspan="2">Description</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr>
-          <th><a href="https://people.canonical.com/~platform/images/xilinx/iot-zcu10x-classic-desktop-2004-x07-20210728-85-sysroot.tar.xz">iot-zcu10x-classic-desktop-2004-x07-20210728-85-sysroot.tar.xz</a></th>
-          <td colspan="2">Sysroot for cross-compilation</td>
-        </tr>
-        <tr>
-          <th><a href="https://people.canonical.com/~platform/images/xilinx/rootfs.ext4.xz">roofts.ext4.xz</a></th>
-          <td colspan="2">EXT4 formatted partition containing entire raw rootfs</td>
-        </tr>
-        <tr>
-          <th><a href="https://people.canonical.com/~platform/images/xilinx/rootfs.tar.gz">rootfs.tar.gz</a></th>
-          <td colspan="2">Raw Root filesystem</td>
-        </tr>
-        <tr>
-          <th><a href="https://people.canonical.com/~platform/images/xilinx/system-boot.tar.gz">system-boot.tar.gz</a></th>
-          <td colspan="2">FAT Partition Contents</td>
-        </tr>
-      </tbody>
-    </table>
+    <div class="p-tabs">
+      <div class="p-tabs__list" role="tablist" aria-label="Install xilinx">
+        <div class="p-tabs__item">
+          <button class="p-tabs__link" role="tab" aria-selected="true" aria-controls="zynq-ultrascale-tab" id="zynq-ultrascale">Xilinx Zynq UltraScale</button>
+        </div>
+        <div class="p-tabs__item">
+          <button class="p-tabs__link" role="tab" aria-selected="false" aria-controls="kira-kv260-vision-ai-tab" id="kira-kv260-vision-ai" tabindex="-1">Xilinx Kria KV260 Vision AI Starter Kit</button>
+        </div>
+      </div>
+
+      <div tabindex="0" role="tabpanel" id="zynq-ultrascale-tab" aria-labelledby="zynq-ultrascale">
+        <div class="row u-vertically-center">
```

Comment:
The top section is indented a little more than the content below it, I think we could remove the padding on this row and its counterpart in the other tab:
```suggestion
        <div class="row u-vertically-center u-no-padding">
```